project_description/readme/USAGE.rst

```diff
@@ -0,0 +1,14 @@
+To set a description for a project, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Edit or create one.
+#. You will see a new *Description* tab, with an HTML area to set the project's description.
+
+.. figure:: static/description/image.png
```

Comment:
I think that the path for the image should be absolute (and not relative) because the `gen-addon-readme` will automatically generate the `static/description/index.html`. See https://github.com/OCA/maintainer-tools/pull/370.

So IMO this line should be:

```
.. figure:: https://raw.githubusercontent.com/OCA/project/12.0/project_description/static/description/image.png
```